cc.newspring.Apollos/Migrations/001_SetupWorkflowsAndTriggers.cs

```diff
@@ -26,10 +26,8 @@
 namespace cc.newspring.Apollos.Migrations
 {
     [MigrationNumber( 1, "1.2.0" )]
```

Comment:
Just a note about versioning, the first number is the plugin migration and the second should be the minimum version of Rock this would run on.  Since most of this depends on the FinancialPayment changes in 1.3.4 we should make a habit to check/change this.